scripts/Deploy-CustomScriptsExtension-v2.ps1

```diff
@@ -150,6 +150,16 @@ else
 # the following keys: name, value, valueType (optional: defaults to string) and their accompanying values
 # the only supported valueTypes are: string (default) and FilePath
 # for FilePath valueType, the path will be tested, content read and base64 encoded before being passed as the parameter value
+
+# we already have some userful parameters that downstream scripts can leverage
+Log-Message "Adding Azure Subscription and AAD related parameters"
+
+$upgradeParameters += @{"name"="azure-resource-group"; "value"=$ResourceGroupName}
```

Comment:
i've been meaning to ask: why are we using powershell? i'd much prefer we stick to python, bash, or ansible wherever possible to better match the technologies edx has adopted. i'm happy to deviate if there's a really good reason, but we should avoid it otherwise